translator/main.py

```diff
@@ -36,6 +37,27 @@
 
 log = logger.get_logger()
 
+SUPPORTED_BUILD_ARGS = [
+    "max_stack",
+    "gc_min_objects",
+    "gc_growth_trigger",
+    "ext_vars",
+    "ext_codes",
+    "tla_vars",
+    "tla_codes",
+    "max_trace",
+]
+SUPPORTED_TRANSLATOR_ARGS = [
```

Comment:
maybe other translator arguments can be added here, like `grafana_label`, `prometheus_label` and `jsonnet_filename`

Reply:
Addressed 8a6a5613aed28b283875ff5dfce6e1a30fa881d6